lib/excon/ssl_socket.rb

```diff
@@ -85,12 +84,12 @@ def initialize(data = {})
       end
 
       if @data[:proxy]
-        request = 'CONNECT ' << @data[:host] << port_string(@data.merge(:omit_default_port => false)) << Excon::HTTP_1_1
+        request = 'CONNECT ' << @data[:host] << port_string(@data.merge('omit_default_port' => false)) << Excon::HTTP_1_1
```

Comment:
I would love to have a symbol instead, but the travis CI builds were not passing with symbols in place here.